tasmota/language/it-IT.h

```diff
@@ -425,113 +425,113 @@
-#define D_KNX_PHYSICAL_ADDRESS_NOTE "( Deve essere univoco nella rete KNX )"
+#define D_KNX_PHYSICAL_ADDRESS "Indirizzo fisico"
+#define D_KNX_PHYSICAL_ADDRESS_NOTE "(deve essere univoco nella rete KNX )"
 #define D_KNX_ENABLE "Abilita KNX"
-#define D_KNX_GROUP_ADDRESS_TO_WRITE "Dati da Inviare al Gruppo di Indirizzi"
+#define D_KNX_GROUP_ADDRESS_TO_WRITE "Dati da inviare al gruppo di indirizzi"
 #define D_ADD "Aggiungi"
 #define D_DELETE "Elimina"
 #define D_REPLY "Rispondi"
-#define D_KNX_GROUP_ADDRESS_TO_READ "Gruppo di Indirizzi da cui Ricevere Dati"
-#define D_RECEIVED_FROM "Ricevuto Da"
+#define D_KNX_GROUP_ADDRESS_TO_READ "Gruppo di indirizzi da cui ricevere dati"
+#define D_RECEIVED_FROM "Ricevuto da"
 #define D_KNX_COMMAND_WRITE "Scrivi"
 #define D_KNX_COMMAND_READ "Leggi"
 #define D_KNX_COMMAND_OTHER "Altro"
 #define D_SENT_TO "invia a"
 #define D_KNX_WARNING "L'indirizzo del gruppo ( 0 / 0 / 0 ) è riservato e non può essere usato."
-#define D_KNX_ENHANCEMENT "Miglioramento Comunicazione"
-#define D_KNX_TX_SLOT "KNX TX"
-#define D_KNX_RX_SLOT "KNX RX"
+#define D_KNX_ENHANCEMENT "Miglioramento comunicazione"
+#define D_KNX_TX_SLOT "KNX - TX"
+#define D_KNX_RX_SLOT "KNX - RX"
 
 // xdrv_03_energy.ino
-#define D_ENERGY_TODAY "Energia Oggi"
-#define D_ENERGY_YESTERDAY "Energia Ieri"
-#define D_ENERGY_TOTAL "Energia Totale"
+#define D_ENERGY_TODAY "Energia - oggi"
+#define D_ENERGY_YESTERDAY "Energia  - ieri"
+#define D_ENERGY_TOTAL "Energia - totale"
 
 // xdrv_27_shutter.ino
-#define D_OPEN "Aperta"
-#define D_CLOSE "Chiusa"
+#define D_OPEN "Apri"
+#define D_CLOSE "Chiudi"
 #define D_DOMOTICZ_SHUTTER "Serranda"
 
 // xdrv_28_pcf8574.ino
 #define D_CONFIGURE_PCF8574 "Configura PCF8574"
 #define D_PCF8574_PARAMETERS "Parametri PCF8574"
-#define D_INVERT_PORTS "Porte Invertite"
+#define D_INVERT_PORTS "Inverti porte"
 #define D_DEVICE "Dispositivo"
 #define D_DEVICE_INPUT "Input"
 #define D_DEVICE_OUTPUT "Output"
 
 // xsns_05_ds18b20.ino
 #define D_SENSOR_BUSY "Sensore occupato"
-#define D_SENSOR_CRC_ERROR "Sensore errore CRC"
+#define D_SENSOR_CRC_ERROR "Errore CRC sensore"
 #define D_SENSORS_FOUND "Sensori trovati"
 
 // xsns_06_dht.ino
-#define D_TIMEOUT_WAITING_FOR "Attesa timeout per"
+#define D_TIMEOUT_WAITING_FOR "Timeout attesa per"
 #define D_START_SIGNAL_LOW "inizio segnale basso"
 #define D_START_SIGNAL_HIGH "inizio segnale alto"
 #define D_PULSE "impulso"
 #define D_CHECKSUM_FAILURE "Checksum fallito"
 
 // xsns_07_sht1x.ino
-#define D_SENSOR_DID_NOT_ACK_COMMAND "Sensore non ha eseguito il comando ACK"
-#define D_SHT1X_FOUND "SHT1X trovato"
+#define D_SENSOR_DID_NOT_ACK_COMMAND "Il sensore non ha eseguito il comando ACK"
+#define D_SHT1X_FOUND "Trovato SHT1X"
 
 // xsns_18_pms5003.ino
 #define D_STANDARD_CONCENTRATION "CF-1 PM"     // Standard Particle CF-1 Particle Matter
 #define D_ENVIRONMENTAL_CONCENTRATION "PM"     // Environmetal Particle Matter
 #define D_PARTICALS_BEYOND "Particelle"
 
 // xsns_32_mpu6050.ino
```

Comment:
Meno caratteri = meno memoria usata.

Reply:
Più caratteri = descrizione più accurata

Reply:
Preferiamo meno memoria usata e la differenza nella descrizione é minima.

Reply:
La differenza di memoria usata dalle stringhe short e normali è infinitesimale in relazione alla memoria disponibile e la chiarezza dovrebbe  esser uno dei punti fondanti di un'interfaccia web.

Le abbreviazioni non servono a nulla se non a rendere poco comprensibile ciò che dovrebbe invece esserlo al massimo.

Reply:
Non ti ho detto di cambiare le stringhe ma ti ho avvisato che meno caratteri per noi significa meno memoria usata. 
Il tuo "infinitesimale"  non é lo stesso per noi che combattiamo tutti i giorni per diminuire la dimensione del codice e far girare tutto in poco più di 25k  di memoria libera.

Reply:
La differenza tra traduzione ultra compressa come testo (=meno chiara) e quelal appena più descrittiva sono pochi byte.

Quanto è lo spazio di memoria disponibile per il codice?

Come ho detto prima all'inizio che queste esperienze (testo corto - etc) siano riportate nel documento iniziale nella home nella sezione che si potrebbe chiamare "Come tradurre - csoa fare e come".

Reply:
Dipendendo dalla configurazione abbiamo un massimo di 30k di memoria disponibilie dove deve girare tutto. Ti assicuro che non é molta cosa. Per farti un esempio avere un a funzione ridotta di 10-15 bytes per noi é motivo di festa :D